topics/introduction/faqs/fetched_data.md

```diff
@@ -0,0 +1,8 @@
+---
+title: I fetched data from a remote website but now I’m logged out of Galaxy and my data is gone?
```

Comment:
Could you mention UCSC here? Otherwise I worry people might miss it.

Reply:
Okay, I will do that!